mtp_noms_ops/templates/security/credits_history_list.html

```diff
@@ -0,0 +1,96 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load mtp_common %}
+{% load security %}
+
+{% block page_title %}
+    {{ view.title }}
+    –
+    {{ block.super }}
+{% endblock %}
+
+
+{% block inner_content %}
+
+<header></header>
+<h1 class="heading-xlarge">{{ view.title }}</h1>
+<aside class="mtp-object-list-toggle print-hidden">
+    <a href="/security/checks/" class="button-secondary">View pending</a>
+</aside></header>{% include 'mtp_common/includes/message_box.html' %}<p class="font-large">
```

Comment:
you ought to have the p element on a new line and the header close tag shouldn't be there